### ISSUE TITLE
Box scores for 2nd week of 2-week playoff matchup are not pulled

### ISSUE THREAD

**Issue by mtdickey** [cwendt94/espn-api#159]
Using `league.box_scores(14)` (week 14 being the most recently completed week, which was week 2 of a 2 week playoff matchup) gives me the following traceback.  This issue was discovered in the previous version and persists after upgrading to 0.9.1.

```
  File "<ipython-input-2-bf17d8a7832e>", line 9, in <module>
    for box in league.box_scores(week):

  File "C:\Users\mtdic\Anaconda3\envs\nfl\lib\site-packages\espn_api\football\league.py", line 209, in box_scores
    box_data = [BoxScore(matchup, pro_schedule, positional_rankings, week, self.year) for matchup in schedule]

  File "C:\Users\mtdic\Anaconda3\envs\nfl\lib\site-packages\espn_api\football\league.py", line 209, in <listcomp>
    box_data = [BoxScore(matchup, pro_schedule, positional_rankings, week, self.year) for matchup in schedule]

  File "C:\Users\mtdic\Anaconda3\envs\nfl\lib\site-packages\espn_api\football\box_score.py", line 7, in __init__
    self.home_score = round(data['home']['rosterForCurrentScoringPeriod']['appliedStatTotal'], 2)

KeyError: 'rosterForCurrentScoringPeriod'
```

**Comment by cwendt94** [comment]
Strange, I am playing in a league that is in the 2nd week of 2-week playoff as well and not running into this problem.

Could you tell me what `league.currentMatchupPeriod` and `league.scoringPeriodId` returns from your league object?

**Comment by mtdickey** [comment]
Hmm, that is interesting.  Here's what I get:
```
In [1]: league.currentMatchupPeriod
Out[1]: 14

In [2]: league.scoringPeriodId
Out[2]: 15
```

**Comment by cwendt94** [comment]
Okay that looks correct and is what my league is returning. I think the only way to find the issue with the data is if you could run the package in debug mode like `league = League(league_id=123, year=2020, debug=True)` and then post the output of the data when calling `league.box_scores()`.

When you initliazte the league in debug mode it will print out a lot of data but you can ignore that. Once you call `box_scores` it will print out the response data on a line with `ESPN API Response:`. That will help me figure out why in the box score a matchup does not have `rosterForCurrentScoringPeriod`.

**Comment by cwendt94** [comment]
I might not need the data. I have an idea on what the issue could be. When getting box score data I am using the current week for `scoringPeriodId` and `filterMatchupPeriodIds` and in your league case those are different, `matchupPeriod = 14` but `scoringPeriodId = 15`.

This shouldn't be to hard to check and the `settings` has a `matchupPeriods` object that maps matchup periods to scoring periods. I am going to try to get this fixed in the next day or two so that it will work for this weekend.

**Comment by dtcarls** [comment]
So which is proper to be passed to `box_scores` as the `week` parameter?

Also, can confirm using latest version and a league from 2019 with a 2 week playoff that league.box_score breaks once `week` equals a week of the playoffs.

This may all be moot if your described method above works, but wanted to give an additional test data point.

```
league = League(league_id=164483, year=2019)

#success
box_scores = league.box_scores(week=13) 

#failure (week 15 and 16 as well) KeyError: 'rosterForCurrentScoringPeriod'
box_scores = league.box_scores(week=14)

**Comment by mtdickey** [comment]
Here's the data that came back.  Hope it helps.  Thank you for the quick response and working on this.
[league_box_score_output.txt](https://github.com/cwendt94/espn-api/files/5704691/league_box_score_output.txt)

**Comment by cwendt94** [comment]
Thanks for the data! So the current week is still fine to pass but I will now need to use the league settings and check the corresponding matchup period to the scoring period. I should be able to work on this tonight/tomorrow.

**Comment by cwendt94** [comment]
Okay, so I believe it is fixed and will be in v0.9.2, one thing to note is the points for the `home` and `away` team will be total points from both weeks.
